quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/CachingPropertyMappers.java

```diff
@@ -56,17 +57,21 @@ public static PropertyMapper<?>[] getClusteringPropertyMappers() {
                     .build(),
               fromOption(CachingOptions.CACHE_EMBEDDED_MTLS_KEYSTORE.withRuntimeSpecificDefault(getDefaultKeystorePathValue()))
                     .paramLabel("file")
+                    .validator(value -> checkCacheEmbeddedMtlsEnabledAndOptionPresent(CachingOptions.CACHE_EMBEDDED_MTLS_KEYSTORE, CachingOptions.CACHE_EMBEDDED_MTLS_KEYSTORE_PASSWORD))
                     .build(),
```

Comment:
```suggestion
                    .mapFrom(HttpOptions.HTTPS_KEY_STORE_FILE)
                    .build(),
```

You can inherit values from the HTTP server if necessary - same as in the `ManagementPropertyMappers` for management port.